launcher/sdw_updater_gui/UpdaterApp.py

```diff
@@ -185,17 +185,17 @@ def run(self):
 
         # apply dom0 state
         self.progress_signal.emit(10)
-        result = Updater.apply_dom0_state()
         # add to results dict, if it fails it will show error message
-        results["apply_dom0"] = result.value
+        results["apply_dom0"] = Updater.apply_dom0_state()
```

Comment:
This is what fixes #685.

Reply:
makes sense since `apply_dom0_state` was already returning the value.